pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -321,6 +428,122 @@ func TestInt_SecurityIntegrations(t *testing.T) {
+	})
+	t.Run("AlterExternalOauth with other options", func(t *testing.T) {
+		_, id, _ := createExternalOauth(t, func(r *sdk.CreateExternalOauthSecurityIntegrationRequest) {
+			r.WithExternalOauthRsaPublicKey(rsaKey).
+				WithExternalOauthRsaPublicKey2(rsaKey)
+		})
+		role1, role1Cleanup := testClientHelper().Role.CreateRole(t)
+		t.Cleanup(role1Cleanup)
+		newIssuer := testClientHelper().Ids.Alpha()
+		setRequest := sdk.NewAlterExternalOauthSecurityIntegrationRequest(id).
+			WithSet(
+				*sdk.NewExternalOauthIntegrationSetRequest().
+					WithEnabled(true).
+					WithExternalOauthIssuer(newIssuer).
+					WithExternalOauthTokenUserMappingClaim([]sdk.TokenUserMappingClaim{{Claim: "bar"}}).
+					WithExternalOauthSnowflakeUserMappingAttribute(sdk.ExternalOauthSecurityIntegrationSnowflakeUserMappingAttributeEmailAddress).
+					WithExternalOauthBlockedRolesList(sdk.BlockedRolesListRequest{BlockedRolesList: []sdk.AccountObjectIdentifier{role1.ID()}}).
+					WithExternalOauthRsaPublicKey(rsaKey).
+					WithExternalOauthRsaPublicKey2(rsaKey).
+					WithExternalOauthAudienceList(sdk.AudienceListRequest{AudienceList: []sdk.AudienceListItem{{Item: "foo"}}}).
+					WithExternalOauthAnyRoleMode(sdk.ExternalOauthSecurityIntegrationAnyRoleModeDisable).
+					WithExternalOauthScopeDelimiter(" ").
+					WithComment("foo"),
+			)
+		err := client.SecurityIntegrations.AlterExternalOauth(ctx, setRequest)
+		require.NoError(t, err)
+
+		details, err := client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertExternalOauth(details, externalOauthDetails{
+			enabled:                                    "true",
+			externalOauthIssuer:                        newIssuer,
+			externalOauthAnyRoleMode:                   string(sdk.ExternalOauthSecurityIntegrationAnyRoleModeDisable),
+			externalOauthRsaPublicKey:                  rsaKey,
+			externalOauthRsaPublicKey2:                 rsaKey,
+			externalOauthBlockedRolesList:              role1.Name,
+			externalOauthAudienceList:                  "foo",
+			externalOauthTokenUserMappingClaim:         "['bar']",
+			externalOauthSnowflakeUserMappingAttribute: string(sdk.ExternalOauthSecurityIntegrationSnowflakeUserMappingAttributeEmailAddress),
+			externalOauthScopeDelimiter:                " ",
+			comment:                                    "foo",
+		})
+
+		unsetRequest := sdk.NewAlterExternalOauthSecurityIntegrationRequest(id).
+			WithUnset(
+				*sdk.NewExternalOauthIntegrationUnsetRequest().
+					WithEnabled(true).
+					WithExternalOauthAudienceList(true),
+			)
+		err = client.SecurityIntegrations.AlterExternalOauth(ctx, unsetRequest)
+		require.NoError(t, err)
+
+		details, err = client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "ENABLED", Type: "Boolean", Value: "false", Default: "false"})
```

Comment:
I'm a bit worried that default values could be changed later on and would cause our tests to fail (mostly I'm referring to `Default` value).

Reply:
We will see that in the tests failing (Snowflake default would have to change) - we have more such tests, I think this is not a big issue for now

Reply:
I can create a ticket to write a custom assert for values returned from `DESCRIBE`. This concerns lots of resources tests.

Reply:
No need resolving